views/facetsWithArticles/clusteredImages/four.hbs

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html>
+	{{#> head title='Clustered Images - v4' subtitle='' }}
```

Comment:
Are some of these stylings shared in the external CSS file?

Reply:
Fixed